src/r2pipe.rs

```diff
@@ -69,6 +69,15 @@ pub trait Pipe {
         }
         Ok(serde_json::from_str(&result)?)
     }
+    ///Escape the command before executing, valid only as of r2 v.5.8.0 "icebucket"
```

Comment:
```suggestion
    /// Escape the command before executing, valid only as of r2 v.5.8.0 "icebucket"
```

Please add spaces after each comment start.